examples/timeWarpExposure.py

```diff
@@ -174,13 +174,13 @@ def run():
                         interpLength,
                     )
                     destWcs = makeWcs(
-                        projName = "TAN",
-                        destCtrInd = destCtrInd,
-                        skyOffset = skyOffset,
-                        rotAng = rotAng,
-                        scaleFac = scaleFac,
-                        srcWcs = srcWcs,
-                        srcCtrInd = srcCtrInd,
+                        projName="TAN",
+                        destCtrInd=destCtrInd,
```

Comment:
Actually, the original version was "correct": https://developer.lsst.io/python/style.html#keyword-assignment-operators-should-be-surrounded-by-a-space-when-statements-appear-on-multiple-lines

Reply:
Yes, but the original version did not pass flake8 linter using our standard published flake8 configuration, which had fewer things disabled than my own settings. I am making the code compliant with our official pep8 settings so that automatic pep8 passes. In summary: our style guide and our official pep8 settings disagree and I'm choosing the pickier standard.